### DISCUSSION TITLE
import source code from watchmaker

### DISCUSSION THREAD

**Discussion by Ragrati** [zepp-health/zeppos-forum#361] [General]
Hi, 
I would like to custom my own diabetic WF
Can I make something here : https://watchface.zepp.com/create
And after, import and add the widget from watchdrip ?
I'm a n00b and I need to make simple but I don't want to ask the developper each time for something
Thanks

**Comment by SashaCX75** [comment]
You can make only the watch face design there. But you will have to add any additional code yourself.
